examples/Transports/FIX/over_one_session.py

```diff
@@ -39,7 +39,7 @@ def fixmsg(source):
     to serialise and parse messages and pass the codec explicitly.
     """
     # python 2 and 3 compatibility
-    source = {tag: val.encode('utf-8') for tag, val in source.items()}
+    # source = {tag: val.encode('utf-8') for tag, val in source.items()}
```

Comment:
This must be wrong - source is now not defined?

Reply:
Oh I see, source is coming from the function signature. In that case, can we remove this line completely instead of commenting it out?

Reply:
removed this line